### PR TITLE
Code refactor

### PR DESCRIPTION
**Pull request by Zhylkaaa**
Separating screens and list item(in porpoise to keep the code clear)

### CODE REVIEW

**Comment by osdnk** [comment]
Thanks. LGTM. It appears that this commit does not change a lot, but modularisation is always a good idea. 👍 

I advise you two things:

First (and maybe consider doing it in this PR) is configuring linter (e.g. eslint). It's a tool for sticking a strict codestyle which enforce to maintain code in one style of programming. It refers to wrapping, line length and aesthetic stuff. Feel free to try it, it's really helpful. I have it configured in every project I work on for for longer than one day :)

Second thing is more high-level rethink of conception of managing code's structure. Maybe try introduce some MVC pattern?

File: customviews/ListItem.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import {StyleSheet, Text, TouchableHighlight, View} from "react-native";
+
+export class ListItem extends React.Component {
+
+    constructor(props){
+        super(props);
```
Comment (osdnk) [inline customviews/ListItem.js]: There's no need for constructor here if it only calls supermethod

File: customviews/ListItem.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    }
+
+    render(){
+        return (
+            <TouchableHighlight onPress={()=>this.props.context.navigation.navigate('Details', {item: this.props.item})}>
```
Comment (osdnk) [inline customviews/ListItem.js]: Try not not define function inline. It provides to redeclaring function (in js is not very cheap) on every rerender or refresh on view. 

But it's not very important:

```
doSth = () => { /* */ }
// or
doSthEls() { /* */ }
// ...
render() {
	return(
		<TouchableOpacity onPres={this.doSth} />
	)
} 


```

File: screens/HomeScreen.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+    getEateries() {
+        fetch("http://localhost:9000/allEateries").then((response) => response.json()).then((responseJson) => {
+            this.setState({
+                isLoading: false,
+                dataSource: responseJson
+            });
+        })
+    }
+
```
Comment (osdnk) [inline screens/HomeScreen.js:27]: unnecessary whitelines (it refers to many places)

**Comment by osdnk** [comment]
@Zhylkaaa Please see my inline comments

**Comment by Zhylkaaa** [comment]
Tried to fix problems with last commit and setup ESLint

File: package.json [hunk 4]
```diff
@@ -13,5 +13,16 @@
     "react": "16.3.1",
     "react-native": "https://github.com/expo/react-native/archive/sdk-30.0.0.tar.gz",
     "react-navigation": "^2.18.2"
+  },
+  "devDependencies": {
+    "eslint": "^5.8.0",
+    "eslint-config-airbnb": "^17.1.0",
```
Comment (osdnk) [inline package.json]: If u use Standard, there's no need to add airbnb

**Review by osdnk — COMMENTED** [review]
Final comment :+1:

**Comment by osdnk** [comment]
cc @Zhylkaaa 
Suppose it does not affect anything, but after removing stuff from `package.json`, you should also update `package-lock.json` (made automagically).

**Pull request merged**